### PR TITLE
feat: display more tree tag

### LINKED ISSUE #705: Tree Page: please display more tree tag on tree page
Please help display the lost tags on tree page, like: age, token, wallet, and so on.

Please show these tags for us (by parsing from the tree API response data):
* status:
* age
* morphology
* species
* token
* wallet

Here is an example of tree API.
https://dev-k8s.treetracker.org/query/trees/186735

```
{"id":186735,"time_created":"2020-10-19T06:47:40.000Z","time_updated":"2020-10-19T06:47:40.000Z","missing":false,"priority":false,"cause_of_death_id":null,"planter_id":940,"primary_location_id":null,"settings_id":null,"override_settings_id":null,"dead":0,"photo_id":null,"image_url":"https://treetracker-dev-images.s3.eu-central-1.amazonaws.com/2020.10.19.09.51.07_-5.5080179325596275_38.981510909625214_bd17cf5e-af94-4644-a02d-ca07d55a02ed_IMG_20201019_094716_5079900695025456370.jpg","certificate_id":null,"estimated_geometric_location":"0101000020E6100000F6C04426A27D4340238058DA350816C0","lat":"-5.5080179325596275","lon":"38.981510909625214","gps_accuracy":3,"active":true,"planter_photo_url":"https://treetracker-dev-images.s3.eu-central-1.amazonaws.com/2020.10.19.09.47.53_-5.508107173727935_38.981361706266256_39f0cc9d-0f13-4547-8142-150f15cabb67_IMG_20201019_094513_6614320100195503436.jpg","planter_identifier":"0784589075","device_id":null,"sequence":null,"note":"","verified":false,"uuid":"a7eb740b-e539-48f2-8b0c-ff77bf2e898d","approved":false,"status":"planted","cluster_regions_assigned":true,"species_id":null,"planting_organization_id":13,"payment_id":null,"contract_id":null,"token_issued":false,"morphology":null,"age":null,"species":null,"capture_approval_tag":null,"rejection_reason":null,"matching_hash":null,"device_identifier":"651f04008af0d91a","images":{},"domain_specific_data":{},"token_id":null,"name":null,"earnings_id":null,"session_id":null}
```

Please choose appropriate icon in our Figma design:
https://www.figma.com/file/XdYFdjlsHvxehlrkPVYq0l/Greenstand-Webmap?node-id=7297%3A36476

### PR DESCRIPTION
**Pull request by pierrelstan**
# Description

The following tree tags have been implemented: 
-  Morphology
-  species 

I only created those two because the **status** and the **wallet** tree tag do not exist in the provided design in Figma and , the tree data does not return any key value for the **wallet**.

resolves #705

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [x] New feature (non-breaking change which adds functionality)

## Screenshots

[comment]: # 'Please include screenshots of your changes if relevant.'

# Checklist:

- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by dadiorchen** [comment]
@pierrelstan awosome!

File: src/components/common/TreeTag.js [hunk 1]
```diff
@@ -56,6 +57,11 @@ function TreeTagComponent({ TreeTagValue, title, icon }) {
             variant="body2"
           >
             {TreeTagValue}
+            {title === 'Age'
+              ? ' Years'
+              : title === 'Diameter at Breast Height'
+              ? ' cm'
+              : ''}
```
Comment (dadiorchen) [inline src/components/common/TreeTag.js:64]: @pierrelstan do you think we can put the logic outside so the tree page will handle the conditions and put raw text into this component? I feel that would relieve the knowledge burden for this component.

Comment (pierrelstan) [reply src/components/common/TreeTag.js:64]: Yes, it's possible. I will make the changes.

File: src/pages/trees/[treeid].js [hunk 2]
```diff
@@ -385,13 +389,30 @@ export default function Tree({
...
         )}
+
+        {tree.morphology && (
+          <TreeTag
+            TreeTagValue={tree.morphology}
+            title="Diameter at Breast Height"
+            icon={<img src={diameterIcon} alt="diameter" />}
+          />
+        )}
+
```
Comment (dadiorchen) [inline src/pages/trees/[treeid].js:415]: @pierrelstan we are unnecessarily making things complicated.

Why don't you just put it here? 

```
<TreeTag
  value={tree.morphology + " cm"}
  title="Diameter at Breast Height"
/>
```

Comment (pierrelstan) [reply src/pages/trees/[treeid].js:415]: Yours is much simpler, thank you for the eye-opening.

**Comment by dadiorchen** [comment]
@pierrelstan great job!

**Pull request merged**